apps/src/templates/teacherDashboard/TeacherDashboardHeader.jsx

```diff
@@ -30,10 +38,27 @@ function TeacherDashboardHeader({
   openEditSectionDialog,
   asyncLoadCourseOfferings,
   isRtl,
+  ageGatedStudentsCount,
+  sectionId,
+  loadSectionStudentData,
 }) {
+  const [modalOpen, setModalOpen] = useState(false);
+
+  const currentUser = getStore().getState().currentUser;
+  const inUSA =
+    ['US', 'RD'].includes(currentUser.countryCode) || !!currentUser.usStateCode;
+  const showAgeGatedStudentsBanner =
+    (inUSA && currentUser.isTeacher && ageGatedStudentsCount > 0) &&
+    DCDO.get('show-age-gated-students-banner', !Rails.env.production?);
```

Comment:
```suggestion
    DCDO.get('show-age-gated-students-banner', false);
```
oops I accidentally put Ruby code in the javascript file.